src/basic/os-util.c

```diff
@@ -193,74 +211,3 @@ int load_extension_release_pairs(const char *root, const char *extension, char *
 
         return load_env_file_pairs(f, p, ret);
 }
-
-int extension_release_validate(
```

Comment:
This was moved when split out of https://github.com/systemd/systemd/pull/18018 when I refactored sysext a bit, since will be needed from for ExtensionImages in pid1 with that PR

Reply:
I'll move it to src/shared then.

Reply:
Updated like that.

Reply:
Thanks!